.github/workflows/openapi.yml

```diff
@@ -22,4 +22,3 @@ jobs:
       # Wait for this issue to be resolved: https://github.com/acacode/swagger-typescript-api/issues/370
       - run: npm run update-schemas && test -z "$(git status --porcelain)"
       - run: npm run build
-      - run: npm run codegen && test -z "$(git status --porcelain)"
```

Comment:
`openapi.yml` might run after `test.yml`, so we may still need to keep this line

Reply:
Sorry, I did not notice that the line is checking codegen. All good 👍